lang/java/protobuf/src/main/java/org/apache/avro/protobuf/ProtobufData.java

```diff
@@ -328,7 +328,7 @@ public Schema getSchema(EnumDescriptor d) {
   private static final ObjectMapper MAPPER = new ObjectMapper(FACTORY);
   private static final JsonNodeFactory NODES = JsonNodeFactory.instance;
 
-  private JsonNode getDefault(FieldDescriptor f) {
+  public JsonNode getDefault(FieldDescriptor f) {
```

Comment:
I'm really not so sure about this one.   For 1.9, we have really tried to make sure any of the methods that directly use  Jackson/JSON are private to not expose that detail.  Making this public would reverse that.

Reply:
That's a good point. I'll revert this one.

Reply:
Reverted the access on `getDefault()`